driverconfig/config.yaml

```diff
@@ -114,14 +114,28 @@ drivers:
         external-health-monitor: gcr.io/k8s-staging-sig-storage/csi-external-health-monitor-controller:v0.6.0
   - name: isilon
     configVersions:
-      - configVersion: v2.6.0
+      - configVersion: v2.7.0
         useDefaults: true
+        supportedVersions:
+          - version: v121
```

Comment:
Do we need to support 1.21 for 2.7 driver?

Reply:
Yes, this is to support MKE-3.5

Reply:
Yes